code/modules/reqs/supplypacks.dm

```diff
@@ -83,7 +83,11 @@ OPERATIONS
 
 /datum/supply_packs/operations/deployablecams
 	name = "3 Deployable Cameras"
-	contains = list(/obj/item/deployable_camera)
+	contains = list(
+		/obj/item/deployable_camera,
+		/obj/item/deployable_camera,
+		/obj/item/deployable_camera,
+		)
```

Comment:
1 less tab :^)

Reply:
Fricken tabs! Ahh

Reply:
Yeah I thought it was the deployablecams needing to be deployable_camera, but no change on my end, your makes more sense.